Push docker images to Docker Hub from CI (#906)
api3dao/airnode: packages/airnode-examples/README.md

[documentation](https://docs.api3.org/airnode/latest/grp-providers/guides/build-an-airnode/configuring-airnode.html) for
more details.

### 9. A Note on Docker Images

API3 makes docker container builds for releases available on

> [Docker Hub](https://hub.docker.com/u/api3/airnode-deployer). API3 also makes development builds available; if this

Reviewer: I suggest removing `airnode-deployer` from this link since it's meant to point to more than just the deployer build.
```suggestion
[Docker Hub](https://hub.docker.com/u/api3/). API3 also makes development builds available; if this
```

repository represents a clone of the clean state of a recent commit it is likely that an image will be available for it
on Docker Hub. In this scenario you will not need to rebuild any containers and can skip container build steps,

> specifically steps 10 and 11.

Reviewer: ```suggestion
specifically steps 10, 11, and 13.
```

This command will utilise the previously built artifacts container to build the containerized version of Airnode which
you can then run locally.

### 14. (Only if running Airnode locally) Run the Airnode container

This command runs Airnode locally. If the local git state is clean and a prebuilt image is available on Docker Hub, this
command will pull the necessary images from Docker Hub.

> If it fails to pull the image, please build the necessary images (refer to step 10).

Reviewer: ```suggestion
If it fails to pull the image, please build the necessary images (refer to steps 10 and 13).
```
